.github/actions/setup-cache/action.yml

```diff
@@ -54,14 +54,13 @@ runs:
       id: contrib-cache
       uses: actions/cache@v4
       with:
-        path: deps/install
+        path: deps
```

Comment:
Originally, dependencies were installed to `deps/install`, is this not the case anymore with the contrib/get-* scripts? Caching the whole deps directory will also include all build artifacts, which can be quite big.

Reply:
> Is this not the case anymore with the contrib/get-* scripts?

That's right.

Notice, however, this `deps` directory is not the one within the `build` directory. This `deps` directory is created at the root, so only artifacts downloaded with the contrib/get-* scripts are put there.

Reply:
Yes, I'm aware of that. I don't think that we want to cache all build artifacts for these dependencies, but only the binaries/libraries we need.

Reply:
Changed. Only `deps/bin` is cached now.